sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncClient.java

```diff
@@ -121,6 +123,7 @@ public final class CosmosAsyncClient implements Closeable {
                                        .withCredential(this.credential)
                                        .withTransportClientSharing(this.enableTransportClientSharing)
                                        .withContentResponseOnWriteEnabled(this.contentResponseOnWriteEnabled)
+                                       .withReadRequestsFallbackEnabled(this.readRequestsFallbackEnabled)
```

Comment:
same here: the readRequestsFallbackEnabled flag has already been populated in connectionPolicy -> .wtihConnectionPolicy